packages/client/hooks/useDraggableReflectionCard.tsx

```diff
@@ -37,17 +37,38 @@ const useRemotelyDraggedCard = (
 ) => {
   const setPortal = useContext(PortalContext)
   const {remoteDrag, isDropping} = reflection
+  const [lastStyle, setLastStyle] = React.useState({})
   const setRemoteCard = (isClose: boolean, timeRemaining: number, lastTop?: number) => {
     if (!drag.ref || timeRemaining <= 0) return
     const beforeFrame = Date.now()
     const bbox = drag.ref.getBoundingClientRect()
     if (bbox.top !== lastTop) {
+      const targetId = remoteDrag?.targetId
       // performance only
-      const style = getDroppingStyles(drag.ref, bbox, windowDims.clientHeight, timeRemaining)
+      const style = getDroppingStyles(
+        drag.ref,
+        bbox,
+        windowDims.clientHeight,
+        timeRemaining,
+        targetId,
+        isClose,
+        lastStyle
+      )
+
+      setLastStyle(style)
```

Comment:
I'm seeing some janky behaviour that might be related to this state update. `requestAnimationFrame` is being executed before every repaint whereas `setState` is asynchronous and might happen in a later frame. 

Is there a way that this could be accomplished without the state update? I'd be happy to look into this with you on Monday if you like!

![remote-anim](https://user-images.githubusercontent.com/39854876/140585073-89fc64f5-dcab-4c7a-8b1e-efb89086c87d.gif)

It also looks like the reflectionCard jumps back to its position on the kanban at the end of the gif below rather than transitioning into its position.

Reply:
@nickoferrall Removing the state seems to have no effect. I think I fixed both issues. Can you check if it works for you?

https://user-images.githubusercontent.com/466991/141094912-d02b8515-2170-4689-b476-4dcffe456f32.mov

Reply:
Beautiful! It's working well for me 👍